app/assets/javascripts/basetypes/Interval.js

```diff
@@ -22,10 +22,17 @@ Interval.prototype.cast = (interval) => {
   }
 
   // Cast to DateTime if it is a string representing a DateTime
-  if (casted.low && Date.parse(casted.low)) {
+  if (casted.low) {
+    if (!Date.parse(casted.low)) {
```

Comment:
could you use the DateTime.cast function here? It does the same Date.parse and error reporting